datafusion/sqllogictest/test_files/string_view.slt

```diff
@@ -718,9 +718,8 @@ EXPLAIN SELECT
 FROM test;
 ----
 logical_plan
-01)Projection: replace(__common_expr_1, Utf8("foo"), Utf8("bar")) AS c1, replace(__common_expr_1, CAST(test.column2_utf8view AS Utf8), Utf8("bar")) AS c2
-02)--Projection: CAST(test.column1_utf8view AS Utf8) AS __common_expr_1, test.column2_utf8view
-03)----TableScan: test projection=[column1_utf8view, column2_utf8view]
+01)Projection: replace(test.column1_utf8view, Utf8("foo"), Utf8("bar")) AS c1, replace(test.column1_utf8view, test.column2_utf8view, Utf8("bar")) AS c2
```

Comment:
I think adding a test that shows this query running (aka run

```sql
SELECT
  REPLACE(column1_utf8view, 'foo', 'bar') as c1,
  REPLACE(column1_utf8view, column2_utf8view, 'bar') as c2
FROM test;
```

in addition to `EXPLAIN`
)

Reply:
Hi @alamb, after updating the replace.rs to process Utf8View, my sql logic test still uses the CAST in its logical plan and this test is failing. Do you have any idea to debug this issue?

Reply:
Hi @alamb, I already found the issue. The reason is that I didn't update the function's signature to Utf8View.

Reply:
> after updating the replace.rs to process Utf8View, my sql logic test still uses the CAST in its logical plan and this test is failing. Do you have any idea to debug this issue?

@thinh2 you're missing define `Utf8View` as probably signature [here](https://github.com/apache/datafusion/blob/9f77021c2e5b10c4ee7e98f1c0887524ebf87ae2/datafusion/functions/src/string/replace.rs#L48)